src/components/Picklist/helpers/isEmpty.js

```diff
@@ -0,0 +1,3 @@
+export default function isEmpty(value) {
+    return [null, undefined, '', {}, []].includes(value);
```

Comment:
when value is {} or [] this always will return false since objects and arrays in javascript are compared by reference not by its value